examples/rendezvous/src/bin/register.rs

```diff
@@ -19,35 +19,37 @@
 // DEALINGS IN THE SOFTWARE.
 
 use futures::StreamExt;
-use libp2p_core::{identity, upgrade::Version, Multiaddr, PeerId, Transport};
-use libp2p_ping as ping;
-use libp2p_rendezvous as rendezvous;
-use libp2p_swarm::AddressScore;
-use libp2p_swarm::{NetworkBehaviour, Swarm, SwarmEvent};
+use libp2p::{
+    core::transport::upgrade::Version,
+    identity, ping, rendezvous,
+    swarm::{derive_prelude, keep_alive, AddressScore, NetworkBehaviour, Swarm, SwarmEvent},
+    Multiaddr, PeerId, Transport,
+};
 use std::time::Duration;
+use void::Void;
 
 #[tokio::main]
 async fn main() {
     env_logger::init();
 
+    let key_pair = identity::Keypair::generate_ed25519();
     let rendezvous_point_address = "/ip4/127.0.0.1/tcp/62649".parse::<Multiaddr>().unwrap();
     let rendezvous_point = "12D3KooWDpJ7As7BWAwRMfu1VU2WCqNjvq387JEYKDBj4kx6nXTN"
         .parse()
         .unwrap();
 
-    let identity = identity::Keypair::generate_ed25519();
-
     let mut swarm = Swarm::with_tokio_executor(
-        libp2p_tcp::tokio::Transport::default()
+        libp2p::tcp::tokio::Transport::default()
             .upgrade(Version::V1)
-            .authenticate(libp2p_noise::NoiseAuthenticated::xx(&identity).unwrap())
-            .multiplex(libp2p_yamux::YamuxConfig::default())
+            .authenticate(libp2p::noise::NoiseAuthenticated::xx(&key_pair).unwrap())
+            .multiplex(libp2p::yamux::YamuxConfig::default())
```

Comment:
We are [in the process](https://github.com/libp2p/rust-libp2p/issues/2217) of apply more consistent naming to all our crates to make usage easier and clearer. Can we import the submodules here please like `use libp2p::tcp;` and then say `tcp::tokio::Transport`?